contrib/cirrus/integration_test.sh

```diff
@@ -49,6 +50,11 @@ else
             ;;
         *) bad_os_id_ver ;;
     esac
-    make localintegration
+    if [[ "$REMOTE" == "true" ]]
+    then
+        make remoteintegration
```

Comment:
Consider adding something like: `echo "***** EXECUTING REMOTE INTEGRATION TESTS *****".

Otherwise this important detail might be easy to loose track of when examining results.

In fact, maybe also add `$REMOTE` to be displayed by `show_env_vars()` function in `contrib/cirrus/lib.sh`.